code/modules/reagents/chemistry/recipes/drinks_reactions.dm

```diff
@@ -1063,3 +1063,11 @@
 	required_reagents = list("space_drugs" = 1, "sodawater" = 1, "grapejuice" = 1)
 	result_amount = 5
 	mix_sound = 'sound/goonstation/misc/drinkfizz.ogg'
+
+/datum/chemical_reaction/planet_cracker
+	name = "Planet Cracker"
+	id = "planet_cracker"
+	result = "planet_cracker"
+	required_reagents = list("fernet" = 10, "voxwine" = 10, "egg" = 2, "gold" = 5)
+	result_amount = 20
+	mix_message = "The liquid's color starts shifting as the nanogold is alternately corroded and redeposited."
```

Comment:
I don't quite remember how we do these on chemical reactions, but as far as I remember we generally only put visual and odor changes in the mix_message?

Reply:
That's a visual change, and I think it's fine to have a bit of flavor in the `mix_message`, especially since it's not like you'll be mixing this every shift.